docs/dg/dev/third-party-module-development/drive-usage-and-support-with-problems.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Drive Usage and Support with Problems
+description: Drive Usage and Support with Problems
+last_updated: Jun 7, 2024
+template: howto-guide-template
+
+---
+
+We would recommend 4 steps how to improve usage of your module:
+1. Provide complete integration guide for you module in **README.md** file.
+   - You can find an example here https://docs.spryker.com/docs/pbc/all/data-exchange/202404.0/install-and-upgrade/install-the-data-exchange-api.html#configure-the-scheduler 
+   - It is important to:
+     - Keep guide up to date for the latest release. (After all the steps executed, the functionality should work for the end user)
+     - Have **Verification** step that clearly states how to check that everything is done properly and functionality works as expected.
+
+2. We recommend to use https://docs.github.com/en/issues/tracking-your-work-with-issues/quickstart for collaboration with module users, mention (In **README.md** file of your module) how issues can be reported.
```

Comment:
Let's write it in a way, that if you do not have a support portal or similar, could use Github issues to interact with the users of your module... smth like that